resources/themes/pterodactyl/layouts/admin.blade.php

```diff
@@ -40,6 +40,17 @@
             <![endif]-->
         @show
     </head>
+    @if(isset($appGoogleAnalyticsId))
```

Comment:
Can we just plop this into another blade template that gets included in these files rather than repeating the same logic throughout them all?